sys-libs/llvm-libunwind/files/Makefile

```diff
@@ -0,0 +1,43 @@
+ASFLAGS+=-Wa,--noexecstack
```

Comment:
Its native build system is cmake, but cmake needs to know the path to llvm's source to configure correctly, although libunwind doesn't depend on any file from llvm.

libcxx also faces the same problem. I personally hate to do it this way, and will see I can fix this cmake nonsense...